client/components/tinymce/plugins/contact-form/dialog/settings.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextValidation from 'components/forms/form-input-validation';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import { validateSettingsToEmail } from './validations';
+
+export default React.createClass( {
+	displayName: 'ContactFormDialogFormSettings',
+
+	propTypes: {
+		to: PropTypes.string,
+		subject: PropTypes.string,
+		email: PropTypes.string,
+		title: PropTypes.string,
+		onUpdate: PropTypes.func.isRequired
+	},
+
+	onToChange( event ) {
+		this.props.onUpdate( { to: event.target.value } );
+	},
+
+	onSubjectChange( event ) {
+		this.props.onUpdate( { subject: event.target.value } );
+	},
+
+	render() {
+		const emailValidationError = ! validateSettingsToEmail( this.props.to );
+
+		return (
+			<div className="editor-contact-form-modal-settings">
+				<SectionHeader label={ this.translate( 'Contact Form Notification Settings' ) } />
+				<Card>
+					<p>
+						{ this.translate(
+							'If you don’t make any changes here, feedback will be sent to the author of the %(postType)s and the subject will be the name of this %(postType)s.',
+							{
+								args: {
+									postType: this.props.postType === 'post' ? this.translate( 'post' ) : this.translate( 'page' )
```

Comment:
Since my last review, we've added some helpers to make post type labels accessible from within the editor, to our goal of enabling custom post type support in the editor (see #3695).

[`<EditorPostType />` is an example of a component displaying labels for custom post types](https://github.com/Automattic/wp-calypso/blob/master/client/post-editor/editor-post-type/index.jsx). You could probably forgo the hardcoding which remains there for post and page (simply as a convenience for bypassing the loading indicator that would otherwise be visible while the editor loads).

All of this assumes we want contact forms to be enabled for custom post types, so correct me if I'm wrong.